ci-operator/step-registry/assisted/vsphere/setup/configure/assisted-vsphere-setup-configure-commands.sh

```diff
@@ -31,9 +31,32 @@ fi
 api_vip=$(jq -r --argjson N 2 --arg PRH "${primaryrouterhostname}" --arg VLANID "${vlanid}" '.[$PRH][$VLANID].ipAddresses[$N]' "${SUBNETS_CONFIG}")
```

Comment:
Maybe move this section inside an `else` block? It seems inefficient to set the value and then immediately change it.

Reply:
Agree, as I don't want to run the rehearsals again and this PR is blocking me, I will do it in a different PR